e2e/wdio/core/tests/form-message.e2e-spec.ts

```diff
@@ -0,0 +1,181 @@
+import { FormMessagePo } from '../pages/form-message.po';
+import {
+    addIsActiveClass,
+    checkElementScreenshot,
+    click, getAttributeByName, getElementArrayLength, getText,
+    getValue,
+    isElementClickable, mouseHoverElement,
+    refreshPage, saveElementScreenshot,
+    scrollIntoView, setValue, waitForPresent
+} from '../../driver/wdio';
+
+import {
+    test,
+    test2,
+    input,
+    button,
+    message,
+    text,
+    message1,
+    message2,
+    message3,
+    message4
+} from '../fixtures/appData/form-message-contents';
+
+import {
+    inputFieldExample,
+    inputFieldHoverSTate,
+    inputFieldActiveState,
+    inputFieldFocusState,
+    buttonExample,
+    buttonHoverState,
+    buttonActiveState,
+    buttonFocusState,
+    messageExample,
+    messageHoverState
+} from '../fixtures/testData/form-message-tags';
+
+describe('Form Message test suite:', function() {
+
+    const formMessagePage = new FormMessagePo();
+    const { messageWithInput, messageWithInputGroup, buttons, messageInformation, messageWithTextArea, inputFields } = formMessagePage;
+
+    beforeAll(() => {
+        formMessagePage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForPresent(messageWithInput);
+    }, 1);
+
+    it('should check message with input field', () => {
+        scrollIntoView(messageWithInput);
+        expect(getAttributeByName(messageWithInput, 'placeholder')).toBe(text);
```

Comment:
Can't we do those checks with unit tests?

Reply:
@nikvalor Please make it a separate check in case we can't